src/sentry/integrations/github/webhook.py

```diff
@@ -73,10 +73,15 @@ def __call__(self, event, host=None):
                 external_id=six.text_type(event['repository']['id']),
             )
             for repo in repos:
-                # We need to track GitHub's "full_name" which is the repository slug.
+                # We need to track GitHub's "full_name" which is the repository
+                # slug (in the form 'github_org_name/repo_name').
                 # This is needed to access the API since `external_id` isn't sufficient.
-                if repo.config.get('name') != event['repository']['full_name']:
-                    repo.config['name'] = event['repository']['full_name']
+                repo_full_name = event['repository']['full_name']
+                # update our data if repo name has changed
+                if repo.config.get('name') != repo_full_name:
+                    repo.config['name'] = repo_full_name
+                    repo.name = repo_full_name
+                    repo.url = u'https://github.com/{}'.format(repo_full_name)
```

Comment:
@MeredithAnya do you happen to know what the correct way to get the domain name for github enterprise? Would it just be the domain name in this case? https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/github_enterprise/integration.py#L335

Reply:
Github sends us a `html_url` as part of the the repository object. Could we use that?

Reply:
@lobsterkatie  @lauryndbrown  ^ what mark said - we should be able to use `event['repository']['html_url']` which will work for both GH and GHE

Reply:
LOL - I actually _had_ that originally, and then wanted to match what we did to configure the integration in the first place. Okay, will go back to it.